MidCamp/midcamp
# Export prod config, add User image alt text, fix Featured Speakers block title

# Description
- Exports active config from prod
- Addresses #137, adding and requiring image alt text for user_picture field
- Fixes bug whereby "Featured Speakers" block view title appeared without content in the view

# To test
- Import config with `drush cim`
- Visit an existing user (or create a new one).  If an image is uploaded, observe you cannot save the entity without adding alt text
- Visit the homepage.  Observe the "Featured Speakers" heading is not displaying.
- Visit the [Featured Speakers entityqueue](http://midcamp.org.docker.amazee.io/admin/structure/entityqueue/featured_speakers/featured_speakers) and add some speakers.  Return to the homepage and observe that the "Featured Speakers" heading displays.
